docker/README.md

```diff
@@ -0,0 +1,48 @@
+# Neurodamus Docker Image
+
+## Requirements
+You must have:
+* [Docker Desktop](https://www.docker.com/) installed and running.
+* [Git](https://git-scm.com/)
+
+## Docker image
+You can either build your own docker image using the recipe in this repository or pull the prebuilt image from the Docker Hub.
+
+### Option 1: Build your docker image
+Before building your own docker image, make sure that your Docker Desktop is configured with at least 4 GB memory and 2 GB swap. (Settings->Resources->Memory)
+```
+cd docker/
+docker build -t neurodamus .
+```
+### Option 2: Pull the prebuilt image from Docker Hub
+```
+docker pull bluebrain/neurodamus
+```
+You can verify your image with the command `docker image ls`, for example:
+```
+$ docker image ls
+REPOSITORY                            TAG       IMAGE ID       CREATED        SIZE
+bluebrain/neurodamus                  latest    4784d73155e7   11 hours ago   4.08GB
+```
+## Docker containier
+With the docker image, you can start a neurodamus container with an interative Bash shell and meanwhile mount your local folder which contains your mod files and the circuit data.
+```
+docker run --rm -it --entrypoint bash -v <folder_mods_circuit>:/mnt/mydata weinaji/neurodamus:0.0.2
```

Comment:
You could probably add an `ENTRYPOINT` instruction at the end of the `Dockerfile` instead of requiring the user to specify it here with `--entrypoint`.

Reply:
Good point. The less user operation the better. I've moved it to the docker file.